src/common/Constants.ts

```diff
@@ -22,22 +22,34 @@ export const DATAWORKER_FAST_LOOKBACK: { [chainId: number]: number } = {
   42161: 691200,
 };
 
-// Reorgs are anticipated on Ethereum and Polygon.
-// Ethereum: Post-merge finality time is 2 epochs (64 slots total), we set the default to 32 which is the
-// length of time for a block to be "justified", which Alchemy (for example) marks as "safe":
-// https://docs.alchemy.com/reference/ethereum-developer-guide-to-the-merge#what-are-safe-and-finalized
-// Polygon: https://polygonscan.com/blocks_forked. There have been re orgs > 128 blocks deep but most CEX's
-// use 128 blocks as the finality time so we'll follow that guideline.
-// Optimistic Rollups are currently centrally serialized and are not expected to reorg. Finality on Optimistic Rollups
-// is technically 1 week, which is too long, and there is little practical difference in
-// finality for any distance between 0 and 1 week.
+// Reorgs are anticipated on Ethereum and Polygon. We use different following distances when processing deposit
+// events based on the USD amount of the deposit.
+// Ethereum: https://etherscan.io/blocks_forked (not working since the merge)
+// Polygon: https://polygonscan.com/blocks_forked
+// Optimistic Rollups are currently centrally serialized and are not expected to reorg.
+export const MIN_DEPOSIT_CONFIRMATIONS_SMALL_TRANSFERS: { [chainId: number]: number } = {
+  1: 4,
+  10: 0,
+  137: 32,
```

Comment:
What if we just had a single MIN_DEPOSIT_CONFIRMATIONS that was divided by threshold? Then we'd just need to modify the code to use the lowest usdVolume key in the object.

It would give us a lot of flexibility in the configuration and doesn't shoehorn us into "small" and "large". For a user who doesn't want to make the distinction, they can just set the limit to 1M (or 0).

```suggestion
export const MIN_DEPOSIT_CONFIRMATIONS: { [usdVolume: number]: { [chainId: number]: number } = {
  5_000: { 1: 4, /* ... */ },
  10_000: { 1: 16, /* ... */ },
  50_000: { 1: 24, /* ... */ },
  1_000_000: { 1: 32, /* ... */ },
}; 
```

Reply:
hmm thats interesting, i'll take a look

Reply:
I've implemented the suggested structure, nice suggestion!

Reply:
I like the new format! Still trying to get my head around the limit structure.